metadata-utils/src/main/java/com/linkedin/metadata/authorization/PoliciesConfig.java

```diff
@@ -130,6 +130,9 @@ public class PoliciesConfig {
   public static final Privilege CREATE_DOMAINS_PRIVILEGE =
       Privilege.of("CREATE_DOMAINS", "Create Domains", "Create new Domains.");
 
+  public static final Privilege CREATE_APPLICATION_PRIVILEGE =
```

Comment:
Do we need to add new privileges? We already have CREATE_ENTITY, EDIT_ENTITY, etc. How is this different?

Reply:
simplified based on david's guidance.